.github/workflows/cypress-mocked.yml

```diff
@@ -34,9 +34,10 @@ jobs:
         env:
           IDAPI_CLIENT_ACCESS_TOKEN: ${{ secrets.IDAPI_CLIENT_ACCESS_TOKEN }}
         with:
-          start: yarn cypress:server
+          start: yarn cypress:mocked:server
```

Comment:
I feel I should probably know this but why do the mocked tests need the IDAPI secret and IAM role?

Reply:
Good question - I don't think they do! I'm going to remove them and see if anything breaks.

Reply:
Ah, I suspect the IAM role lets the server actually fetch the config it needs to run from S3.

Reply:
Have removed the IDAPI secret and the mocked tests work just fine.